bioptim/limits/path_conditions.py

```diff
@@ -632,6 +632,109 @@ def __init__(
         super(QAndQDotBounds, self).__init__(min_bound=x_min, max_bound=x_max)
 
 
+class XiaFatigueStateBounds(Bounds):
+    """
+    Specialized Bounds that reads a model to automatically extract fatigue state bounds
+    """
+
+    def __init__(
+        self,
+        biorbd_model,
+        has_muscles: bool = False,
+        has_torque: bool = False,
+    ):
+        """
+        Parameters
+        ----------
+        biorbd_model: biorbd.Model
+            A reference to the model
+        has_muscles: bool
+            Applying fatigue on muscles or not
+        has_torque: bool
+            Applying fatigue on torques or not
+        """
+        if has_torque is False:
+            if has_muscles is False:
```

Comment:
This seems to do : 
`if not has_torque and not has_muscles:`
but in a sketchy way. If so, rewrite. Also, `is` for testing the truthiness can result in undefined behavior. Please use `not`